dlp/snippets/deid/deid_test.go

```diff
@@ -117,3 +119,54 @@ func TestDeidentifyExceptionList(t *testing.T) {
 	}
 
 }
+
+func TestDeidentifyTableBucketing(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	row1 := &dlppb.Table_Row{
```

Comment:
comment: Remove these tables from the test.

It's sufficient to embed the table in the sample, as it exists for demonstration purposes. That way customers know how to construct a table.

Reply:
Done.